tests/smoke_test.go

```diff
@@ -452,3 +453,45 @@ func TestPluginRun(t *testing.T) {
 	_, stderr = e.RunCommandExpectError("pulumi", "plugin", "run", "--kind=resource", "random", "--", "--help")
 	assert.Contains(t, stderr, "flag: help requested")
 }
+
+func TestInstall(t *testing.T) {
+	t.Parallel()
+
+	for _, runtime := range Runtimes {
+		// Reassign runtime before capture since it changes while looping.
+		runtime := runtime
+
+		t.Run(runtime, func(t *testing.T) {
+			t.Parallel()
+
+			e := ptesting.NewEnvironment(t)
+			defer deleteIfNotFailed(e)
+
+			// Make sure the random provider is not installed locally
+			// so that we can test the `install` command works.
+			out, _ := e.RunCommand("pulumi", "plugin", "ls")
+			if strings.Contains(out, "random  resource") {
+				e.RunCommand("pulumi", "plugin", "rm", "resource", "random", "--all", "--yes")
+			}
```

Comment:
Sorry for not noticing this before, but I think this could also go away now.  Since we have a fresh `PULUMI_HOME` it's never going to have the plugin.

Reply:
It can't hurt though right? might be best in case something changes for some unknown reason?

Reply:
I mean it adds some time to the test, and will probably be parrotted in other tests as well.  But I don't feel very strongly about it.